src/components/SearchJob.jsx

```diff
@@ -3,23 +3,69 @@ import {
 	Form,
 } from 'react-bootstrap';
 import cn from 'classnames';
+import styled from 'styled-components';
 
 function SearchJob() {
+	const FormWrapper = styled.div`
```

Comment:
Don't define these inside the component. Do it at the root level

Reply:
i changed this